### PR TITLE
###  { Delete empty sections}

### PR DESCRIPTION
**Pull request by AlexHmelevskiAG**
Changes:
* added the flag to delete empty sections on the table view

### CODE REVIEW

File: EZSource/Classes/TableViewDataSource/SectionSource.swift [hunk 1]
```diff
@@ -65,4 +65,20 @@ class SectionSource {
         self.sections = sections.replacingOccurrences(with: section, where: { $0.id == section.id })
     }
     
+    func deleteEmptySections() -> [DeleteSectionInfo] {
```
Comment (piyush9290) [inline EZSource/Classes/TableViewDataSource/SectionSource.swift]: Shouldn't this be public? I still wondered how it's accessible in Example

Comment (beljamali) [inline EZSource/Classes/TableViewDataSource/SectionSource.swift]: here there is no delete, maybe we have to rename it

Comment (AlexHmelevskiAG) [reply EZSource/Classes/TableViewDataSource/SectionSource.swift]: It still deletes the section in SectionSource and then the info used to delete them on the TV level

Comment (AlexHmelevskiAG) [reply EZSource/Classes/TableViewDataSource/SectionSource.swift]: I guess we resolved this during the discussion?

Comment (beljamali) [reply EZSource/Classes/TableViewDataSource/SectionSource.swift]: great thank you

**Pull request merged**